ifopt_ipopt/include/ifopt/ipopt_solver.h

```diff
@@ -57,13 +57,18 @@ class IpoptSolver : public Solver {
     */
   void Solve(Problem& nlp) override;
 
-  /** Options for the IPOPT solver. A complete list can be found here:
+  /** Set options for the IPOPT solver. A complete list can be found here:
     * https://www.coin-or.org/Ipopt/documentation/node40.html
     */
   void SetOption(const std::string& name, const std::string& value);
   void SetOption(const std::string& name, int value);
   void SetOption(const std::string& name, double value);
 
+  /** Get options of the IPOPT solver.
```

Comment:
These two functions are not used if i'm not mistaken? so let's better remove them (and the .cc implementation) until we notice we need this call more often.

Reply:
You are right, they are not used anymore. I implemented them to read the solver options, but they turned out not to be necessary. I removed them from the .h and .cc files.